fedocal/__init__.py

```diff
@@ -621,7 +621,12 @@ def ical_calendar_meeting(meeting_id):
     if not meeting:
         return flask.abort(404)
     ical = vobject.iCalendar()
-    fedocallib.add_meeting_to_vcal(ical, meeting)
+    fedocallib.add_meeting_to_vcal(
+        ical, meeting,
+        enable_reminder=flask.request.args.get('reminder') == '1',
```

Comment:
``['1', 'true']`` makes sense. I will update the code to handle both the values.